engine/executor/dm/worker.go

```diff
@@ -141,6 +144,14 @@ func (w *dmWorker) InitImpl(ctx context.Context) error {
 		}
 	}
 	w.cfg.MetricsFactory = w.MetricFactory()
+	w.stageGauge = w.MetricFactory().NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: "dataflow",
```

Comment:
not sure what's the namespace

/cc @CharlesCheung96

Reply:
Perhaps dm is a more appropriate namespace because `task-state` is a business-scope monitor.